index2.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+   
+    <link rel="stylesheet" href="styles2.css">
+</head>
+<body>
+    <!--For small screen, this will be display when the screen is less than 768px-->
+    <header id="smallScreen">
+        <a href="">Welcome</a>
+        <nav>
+            <div class="c1"> </div>
+            <div class="c2"> </div>
+            <div class="c3"> </div>
+        </nav>
+    </header>
+    <main id="main">
+        <!--For big screen, this will be hide when screen is less than 768px(just navigation)--> 
+        <nav class="navigation">
+            <ul>
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        
+        <div class="headline">
+            <header class="header">
+                <h1>Hey there. I’m Franck sefu.<br>
+                <span>I’m a software developer</span></h1>
+            </header>
+            <section class="content">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
```

Comment:
- [ ] The previous reviewer mention this p-tag design issue, please update your color to match this original design

<img width="805" alt="1" src="https://user-images.githubusercontent.com/77843050/212210925-0af409ff-ddbf-43b3-9391-c9f3a33f4e9b.png">